build/release.mk

```diff
@@ -151,6 +151,7 @@ package-general:
 	mkdir -p $(DIST_PATH_GENERIC)/bin
 	mkdir -p $(DIST_PATH_GENERIC)/logs
 	mkdir -p $(DIST_PATH_GENERIC)/prepackaged_plugins
+	mkdir -p $(DIST_PATH_GENERIC)/products
```

Comment:
Shouldn't this be `/client/products`?

Reply:
Right. That's left over from the old version. It's not actually needed since we're doing a `mkdir -p` when copying the Boards files anyway